monitoring/opentelemetry-reactive/src/test/java/io/quarkus/ts/opentelemetry/reactive/OpenShiftGrpcIT.java

```diff
@@ -0,0 +1,42 @@
+package io.quarkus.ts.opentelemetry.reactive;
+
+import io.fabric8.kubernetes.api.model.Container;
+import io.fabric8.kubernetes.api.model.GRPCAction;
+import io.fabric8.kubernetes.api.model.Pod;
+import io.fabric8.kubernetes.api.model.Probe;
+import io.quarkus.test.bootstrap.inject.OpenShiftClient;
+import io.quarkus.test.scenarios.OpenShiftDeploymentStrategy;
+import io.quarkus.test.scenarios.OpenShiftScenario;
+import io.quarkus.test.scenarios.annotations.DisabledOnQuarkusVersion;
+import jakarta.inject.Inject;
+import org.junit.jupiter.api.Assertions;
+import org.junit.jupiter.api.Test;
+
+import java.util.List;
+
+@OpenShiftScenario(deployment = OpenShiftDeploymentStrategy.UsingOpenShiftExtension)
+@DisabledOnQuarkusVersion(version = "2\\..*", reason = "GRPC probes are used only in Quarkus 3")
+public class OpenShiftGrpcIT extends OpenTelemetryGrpcIT {
+
+    @Inject
+    static OpenShiftClient oc;
+
+    @Test
+    void check() {
```

Comment:
please rename this test method so that it is clear from name what does it check (e.g. in Polarion without reading code)?